### PR TITLE
Add NER TF2 example.

### PR DESCRIPTION
**Pull request by jplu**
Create a NER example similar to the Pytorch one. It takes the same options, and can be run the same way.

As you asked @julien-c I prefered I did a fresh new PR :)

### CODE REVIEW

File: transformers/__init__.py [hunk 1]
```diff
@@ -162,6 +162,7 @@
     from .modeling_tf_distilbert import (TFDistilBertPreTrainedModel, TFDistilBertMainLayer,
                                          TFDistilBertModel, TFDistilBertForMaskedLM,
                                          TFDistilBertForSequenceClassification,
+					 TFDistilBertForTokenClassification,
```
Comment (julien-c) [inline transformers/__init__.py]: whitespace issue

Comment (jplu) [reply transformers/__init__.py]: Should be ok now

**Comment by thomwolf** [comment]
This looks great, amazing work @jplu!

Before merging we would need to add:
- a few tests on the optimizer (like in `./transformers/tests/optimization_test.py`)
- documentation for the optimizer (for instance in `./docs/source/main_classes/optimizer_schedules.rst.py`)
- an example of a command line to run the `run_tf_ner.py` script and the associated results you should obtain (in `./examples/README.md`)

Do you think you can do it?

**Comment by jplu** [comment]
Thanks a lots! :)

I can do theses tasks, no problems!!

**Comment by jplu** [comment]
I have done what you asked @thomwolf, please let me know if I have to change something.

**Review by LysandreJik — APPROVED** [review]
This is great! Love that you're using a strategy as well as TF Records, we didn't have any example in the library showcasing the use of those before.

**Comment by thomwolf** [comment]
This is awesome, merging!

**Comment by jplu** [comment]
Amazing!! Thanks a lot ;)

**Pull request merged**